docker-compose.yml

```diff
@@ -8,26 +8,29 @@ services:
             - ./config/DHIS2_home/dhis.conf:/DHIS2_home/dhis.conf
             - ./config/server.xml:/usr/local/tomcat/conf/server.xml
             - home:/DHIS2_home
-            - data:/DHIS2_home/files/
+            - ./config/dhis2-core-start.sh:/usr/local/bin/dhis2-core-start.sh
+            - data:/DHIS2_home/data
             - "${SCRIPTS_DIR}:/DHIS2_home/scripts"
         environment:
-            CATALINA_OPTS: "-Dcontext.path='${DHIS2_CORE_CONTEXT_PATH}'"
+            CATALINA_OPTS: "-Dcontext.path="
+        command: sh /usr/local/bin/dhis2-core-start.sh
```

Comment:
Cool!

Reply:
We are near, but it will be necessary to override the `entrypoint` also, since this command is run as `tomcat`, where you cannot 1) Install curl, 2) Overwrite tomcat files in pre/post shell scripts.